jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -255,6 +255,50 @@ bool ci_lazy_methodst::operator()(
   return false;
 }
 
+// Look for virtual callsites with no candidate targets. If we have
+// invokevirtual A.f and we don't believe either A or any of its children
+// may exist, we assume specifically A is somehow instantiated. Note this
+// may result in an abstract class being classified as instantiated, which
+// stands in for some unknown concrete subclass: in this case the called
+// method will be a stub.
+void ci_lazy_methodst::insert_virtual_methods(
```

Comment:
Possibly GH being weird - but I can't see this method being used anywhere in this PR (or where it is being extracted from?)

Reply:
Not github's fault, either I lost a commit or the refactoring forgot to replace the old code by a call to this function.
I corrected that.